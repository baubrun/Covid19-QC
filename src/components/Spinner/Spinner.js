import React from "react";
import Loader from "react-loader-spinner";
import "./Spinner.css"


const Spinner = () => {
  return (
    <div className="Spinner">
      <Loader color="white" height={200} type="Puff" width={200} />
    </div>
  );
};

export default Spinner;
