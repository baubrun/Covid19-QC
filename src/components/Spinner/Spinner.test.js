import React from "react";
import Spinner from "./Spinner";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("Spinner Component", () => {


  it("should match title ", () => {
    const component = shallow(<Spinner />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
