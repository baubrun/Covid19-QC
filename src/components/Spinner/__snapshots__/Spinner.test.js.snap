// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component should match snapshot  1`] = `
<div
  className="Spinner"
>
  <Loader
    className=""
    color="white"
    height={200}
    style={Object {}}
    timeout={0}
    type="Puff"
    visible={true}
    width={200}
  />
</div>
`;
